adobe_vipm/flows/validation/transfer.py

```diff
@@ -141,30 +141,47 @@ def add_lines_to_order(
 
     order_error = False
 
-    returned_skus = [get_partial_sku(item["offerId"]) for item in adobe_items]
-    items = get_product_items_by_skus(
-        mpt_client, order["agreement"]["product"]["id"], returned_skus
-    )
-    one_time_skus = [
-        item["externalIds"]["vendor"]
-        for item in items
-        if item["terms"]["period"] == "one-time"
-    ]
-    adobe_items_without_one_time_offers = [
-        item
-        for item in adobe_items
-        if get_partial_sku(item["offerId"]) not in one_time_skus
-    ]
-
-    if is_transferred:
-        if are_all_transferring_items_expired(adobe_items_without_one_time_offers):
-            # If the order already has items and all the items on Adobe to be migrated are
-            # expired, the user can add, edit or delete the expired subscriptions
-            if len(order["lines"]):
-                return False, order
-
-            adobe_items = adobe_items_without_one_time_offers
+    if adobe_items:
```

Comment:
I assumed that `adobe_items` may be an empty list in certain cases. I need someone who knows the system better to validate that assumption.